recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ package:
 source:
   - url: https://codeload.github.com/opencv/opencv/tar.gz/{{ version }}
     fn: opencv-{{ version }}.tar.gz
-    sha256: 77f616ae4bea416674d8c373984b20c8bd55e7db887fd38c6df73463a0647bab
+    sha256: a1cfdcf6619387ca9e232687504da996aaa9f7b5689986b8331ec02cb61d28ad
     patches:
       - 20386-SetTotalBytesLimit-one-arg.patch
   - url: https://codeload.github.com/opencv/opencv_contrib/tar.gz/{{ version }}
```

Comment:
I think the contrib sha256 needs to be updated too.

Reply:
Done in https://github.com/conda-forge/opencv-feedstock/pull/296/commits/6396c6d3417b8c4a82e7759c52d4f62943cdb174 .